core/pkg/eval/fractional_evaluation.go

```diff
@@ -58,7 +58,7 @@ func parseFractionalEvaluationData(values, data interface{}) (string, []fraction
 
 	valueToDistribute, ok := v.(string)
 	if !ok {
-		return "", nil, fmt.Errorf("var: %s isn't of type string", bucketBy)
+		return "", nil, fmt.Errorf("var: %s isn't of type string", v)
```

Comment:
maybe it is better to use the default format since we know it is not a string
```suggestion
		return "", nil, fmt.Errorf("var: %v isn't of type string", v)
```